backend/src/main/java/gov/cdc/usds/simplereport/config/DataHubClientConfiguration.java

```diff
@@ -33,7 +36,13 @@ public RequestInterceptor headerRequestInterceptor() {
         template.body("{}");
       }
 
-      template.header("client", simpleReportCsvUploadClientName);
+      var contentType = template.headers().get("Content-Type");
+      if (contentType != null && contentType.contains("application/fhir+ndjson")) {
+        template.header("client", simpleReportCsvUploadFhirClientName);
+      } else {
+        template.header("client", simpleReportCsvUploadClientName);
```

Comment:
Can you pull this out to a function
`template.header("client", getClientName(template.headers().get("Content-Type")));`